<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HudzClass</class>
 <widget class="QMainWindow" name="HudzClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hudz</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="GBox_Sever">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="8" column="2">
        <widget class="QLineEdit" name="LEdit_Ping">
         <property name="text">
          <string>Wait...</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="Label_MainSever">
         <property name="text">
          <string>MainServer</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="Label_ConnectState">
         <property name="text">
          <string>ConnectState</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="LEdit_IpAddress">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="LEdit_Heat">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Label_IpAddress">
         <property name="text">
          <string>IpAddress</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="Label_Heat">
         <property name="text">
          <string>Heat</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="LEdit_IpPort">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="LEdit_MainSever">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="Label_MaintainTime">
         <property name="text">
          <string>MaintainTime</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="Label_Ping">
         <property name="text">
          <string>Ping</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="CBox_Sever_list"/>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="LEdit_ConnectState">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="LEdit_MaintainTime">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Label_Sever">
         <property name="text">
          <string>Sever</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="LEdit_ZoneName">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Label_IpPort">
         <property name="text">
          <string>IpPort</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Label_ZoneName">
         <property name="text">
          <string>ZoneName</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QGroupBox" name="GBox_JoinList">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="7" column="1">
           <widget class="QPushButton" name="PButton_Pause">
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="Label_CurrentTime">
            <property name="text">
             <string>CurrentTime</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="PButton_Start">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="SBox_RepeatTimes">
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="Label_TargetTime">
            <property name="text">
             <string>TargetTime</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="Label_RepeatTimes">
            <property name="text">
             <string>RepeatTimes</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateTimeEdit" name="DTEdit_Target">
            <property name="displayFormat">
             <string>yyyy/M/d H:mm:ss</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateTimeEdit" name="DTEdit_Current">
            <property name="displayFormat">
             <string>yyyy/M/d H:mm:ss</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="Label_Delay">
            <property name="text">
             <string>Delay</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="SBox_Delay">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>350</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hudz.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>PButton_Start</sender>
   <signal>clicked()</signal>
   <receiver>HudzClass</receiver>
   <slot>PButtonStartClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PButton_Pause</sender>
   <signal>clicked()</signal>
   <receiver>HudzClass</receiver>
   <slot>PButtonPauseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>PButtonStartClicked()</slot>
  <slot>PButtonPauseClicked()</slot>
 </slots>
</ui>
